content/pt-br/docs/concepts/security/pod-security-policy.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Políticas de Segurança do Pod
+content_type: conceito
```

Comment:
content_type doesn't needs to be translated so conceito -> concept

Reply:
done